components/ai_chat/resources/page/components/input_box/index.tsx

```diff
@@ -76,10 +78,21 @@ function InputBox(props: InputBoxProps) {
     }
   }
 
+  const querySubmitted = React.useRef(false)
+
   const maybeAutofocus = (node: HTMLTextAreaElement | null) => {
-    if (node && props.context.selectedActionType) {
+    if (!node) {
+      return
+    }
+
+    if (props.context.selectedActionType) {
       node.focus()
     }
+    if (props.context.isMobile && props.context.hasAcceptedAgreement &&
+      !props.context.hasInitialHistory && !querySubmitted.current) {
+        node.focus()
+        getPageHandlerInstance().pageHandler.handleShowSoftKeyboard()
```

Comment:
I think this should go through the `context` so we can bind the `input_box` to another mojom interface (which I think we do for the AI Rewriter stuff, which currently isn't active)